clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java

```diff
@@ -190,8 +190,8 @@
             + " ordering of records because if two batches are sent to a single partition, and the first fails and is retried but the second"
             + " succeeds, then the records in the second batch may appear first. Note additionall that produce requests will be"
             + " failed before the number of retries has been exhausted if the timeout configured by"
-            + " " + DELIVERY_TIMEOUT_MS_CONFIG + " expires first before successful acknowledgement. Users should generally"
-            + " prefer to leave this config unset and instead use " + DELIVERY_TIMEOUT_MS_CONFIG + " to control"
+            + " <code>" + DELIVERY_TIMEOUT_MS_CONFIG + "</code> expires first before successful acknowledgement. Users should generally"
```

Comment:
Do we need a space after "by"?

Reply:
There's a leading space on the next line, before `<code>`.

Reply:
Ah, you're right. Not sure how I missed that.